recipe/0001-Use-entry_points.patch

```diff
@@ -0,0 +1,52 @@
+From ec26151b9e757b95bd00a9a0e068a86d6f85726e Mon Sep 17 00:00:00 2001
+From: BastianZim <10774221+BastianZim@users.noreply.github.com>
+Date: Tue, 30 Nov 2021 19:46:11 +0100
+Subject: [PATCH] Use entry_points
+
+---
+ setup.py                            | 8 ++++++--
+ src/antlr4/commands/__init__.py     | 1 +
+ {bin => src/antlr4/commands}/pygrun | 0
+ 3 files changed, 7 insertions(+), 2 deletions(-)
+ create mode 100644 src/antlr4/commands/__init__.py
+ rename {bin => src/antlr4/commands}/pygrun (100%)
+
+diff --git a/setup.py b/setup.py
+index 75d0dc5..10ca2a7 100644
+--- a/setup.py
++++ b/setup.py
+@@ -3,7 +3,7 @@ from setuptools import setup
+ setup(
+     name='antlr4-python3-runtime',
+     version='4.9.3',
+-    packages=['antlr4', 'antlr4.atn', 'antlr4.dfa', 'antlr4.tree', 'antlr4.error', 'antlr4.xpath'],
++    packages=['antlr4', 'antlr4.atn', 'antlr4.commands', 'antlr4.dfa', 'antlr4.tree', 'antlr4.error', 'antlr4.xpath'],
+     package_dir={'': 'src'},
+     install_requires=[
+         "typing ; python_version<'3.5'",
+@@ -12,6 +12,10 @@ setup(
+     license='BSD',
+     author='Eric Vergnaud, Terence Parr, Sam Harwell',
+     author_email='eric.vergnaud@wanadoo.fr',
+-    scripts=["bin/pygrun"],
++    entry_points={
++         'console_scripts': [
++             'wsdump=websocket.commands.wsdump:main',
```

Comment:
should probably terminate in `pygrun=antlr4.commands.pygrun`

Reply:
Oh shoot sorry forgot to push that.